katdal/visdatav4.py

```diff
@@ -185,8 +185,12 @@ def _before(date):
             src_stream = attrs['src_streams'][0]
             # XXX: should use telstate.join if attrs is a telstate
             self.cbf_dump_period = attrs[src_stream + '_int_time']
+            cbf_n_accs = attrs[src_stream + '_n_accs']
         except (KeyError, IndexError):
-            self.cbf_dump_period = None
+            self.cbf_dump_period = self.accumulations_per_dump = None
+        else:
+            cbf_dumps_per_sdp_dump = int(round(self.dump_period / self.cbf_dump_period))
```

Comment:
```suggestion
            cbf_dumps_per_sdp_dump = round(self.dump_period / self.cbf_dump_period)
```

I discovered recently that in Python 3, `round` already returns an `int`.

Reply:
Thanks, fixed!